restli-common/src/main/pegasus/com/linkedin/restli/restspec/RestMethodSchema.pdl

```diff
@@ -10,6 +10,18 @@ record RestMethodSchema includes CustomAnnotationSchema, ServiceErrorsSchema, Su
    */
   method: string
 
+  /**
+   * Java method type for this rest method
+   */
+  javaMethodType: optional JavaMethodType = "SYNC"
+
+  /**
+   * True if the java implementation of this method returns the result wrapper (aka CollectionResult for GET_ALL, or
+   * GetResult for GET or BatchResult for Batch_GET). False, if it returns the result as is. This is only applicable
+   * for read methods (GET/BATCH_GET/GET_ALL).
+   */
+  javaReturnsResultWrapper: optional boolean
```

Comment:
For difference btw CreateResponse and CreateKVResponse, BatchCreateResult vs BatchCreateKVResult, are we planning to determine that based on returnEntity annotation?

Reply:
Correct. That is already in the IDL.